lib/matplotlib/_mathtext.py

```diff
@@ -452,7 +451,7 @@ class UnicodeFonts(TruetypeFonts):
     This class will "fallback" on the Bakoma fonts when a required
     symbol can not be found in the font.
     """
-    use_cmex = True
+    use_cmex = True  # Deprecated.
```

Comment:
I think this is more helpful for us in the future:
```suggestion
    use_cmex = True  # Unused. Delete once matplotlib.mathtext is private.
```
Same comment should be used below as well.

Reply:
yup